peachjam/templates/peachjam/layouts/main.html

```diff
@@ -24,13 +24,17 @@
     {% if PEACHJAM_SETTINGS.google_analytics_id %}
       <!-- Global site tag (gtag.js) - Google Analytics -->
       <script async
-              src="https://www.googletagmanager.com/gtag/js?id={{ PEACHJAM_SETTINGS.google_analytics_id }}"></script>
+              src="https://www.googletagmanager.com/gtag/js?id={{ PEACHJAM_SETTINGS.google_analytics_id|split|first }}"></script>
       <script>
         window.dataLayer = window.dataLayer || [];
         function gtag(){dataLayer.push(arguments);}
         gtag('js', new Date());
-
-        gtag('config', '{{ PEACHJAM_SETTINGS.google_analytics_id }}');
+      // split multiple ids into array
+      const google_analytics_ids = '{{ PEACHJAM_SETTINGS.google_analytics_id }}'.split(/\s+/).map(id => id.trim());
```

Comment:
```suggestion
      const google_analytics_ids = '{{ PEACHJAM_SETTINGS.google_analytics_id }}'.trim().split(/\s+/));
```

that'll handle all cases. Don't discount how admins can get whitespace and other things wrong.

Reply:
```
" a   b   a  c " .trim().split(/\s+/)
(4) ['a', 'b', 'a', 'c']
```